### PR TITLE
Improve documentation about Express version

### PR DESCRIPTION
**Pull request by A-312**
Remove 3.X and add "Supported Express version" section.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -110,9 +115,9 @@ cons.swig('views/page.html', { user: 'tobi' })
 
  To enable caching simply pass `{ cache: true }`. Engines _may_ use this option to cache things reading the file contents, compiled `Function`s etc. Engines which do _not_ support this may simply ignore it. All engines that consolidate.js implements I/O for will cache the file contents, ideal for production environments.
  When using consolidate directly: `cons.swig('views/page.html', { user: 'tobi', cache:true }, callback);`
- Using Express 3 or higher: `app.locals.cache = true` or set NODE_ENV to 'production' and Express will do this for you.
+ Using Express or higher: `app.locals.cache = true` or set NODE_ENV to 'production' and Express will do this for you.
```
Comment (doowb) [inline Readme.md]: I think this should say `Using support Express versions:` instead of `Using Express or higher:`

**Comment by doowb** [comment]
Thanks, I made a comment that should be updated, then I think it looks good.

**Comment by A-312** [comment]
@doowb ready

**Comment by doowb** [comment]
Thanks!

**Pull request merged**